libs/api/domains/application/src/lib/application.resolver.ts

```diff
@@ -163,4 +164,12 @@ export class ApplicationResolver {
   ): Promise<Application> {
     return this.applicationService.uploadSignedFile(input, user.authorization)
   }
+
+  @Query(() => Application, { nullable: true })
+  getPresignedUrl(
+    @Args('input') input: GetPresignedUrlInput,
+    @CurrentUser() user: User,
+  ): Promise<Application> {
```

Comment:
Is this returning the `Application` object? or a presigned URL?

Reply:
It returns an update application which contains the presigned url as an attachment. 
The reason why I went for that way was that I noticed that every endpoint in the AS returns an application. So I thought that was the architecture you were going for. Is that wrong? Should I return just the `Promise<string>`?

Reply:
I'm not sure about it, but it might be an architectural decision indeed. In my opinion, it looks a bit weird to have the wording of "getUrl" and you get an application in return where you have to look for the field you need but that's just my 2ct

Reply:
Agree and fixed, I also fixed it for other endpoints so the PR is a little bit bigger 😅